Fix node integration tests

Node tests are failing because some module is not found. Check here: https://travis-ci.com/src-d/go-mysql-server/jobs/241958315

```
  Error thrown in test:
  Error {
    code: 'MODULE_NOT_FOUND',
    requireStack: [
      '/home/travis/build/src-d/go-mysql-server/_integration/javascript/node_modules/mysql/lib/protocol/Parser.js',
      '/home/travis/build/src-d/go-mysql-server/_integration/javascript/node_modules/mysql/lib/protocol/Protocol.js',
      '/home/travis/build/src-d/go-mysql-server/_integration/javascript/node_modules/mysql/lib/Connection.js',
      '/home/travis/build/src-d/go-mysql-server/_integration/javascript/node_modules/mysql/index.js',
      '/home/travis/build/src-d/go-mysql-server/_integration/javascript/test.js',
      '/home/travis/build/src-d/go-mysql-server/_integration/javascript/node_modules/ava/lib/worker/subprocess.js',
    ],
    message: `Cannot find module 'bignumber.js'␊
    Require stack:␊
    - /home/travis/build/src-d/go-mysql-server/_integration/javascript/node_modules/mysql/lib/protocol/Parser.js␊
    - /home/travis/build/src-d/go-mysql-server/_integration/javascript/node_modules/mysql/lib/protocol/Protocol.js␊
    - /home/travis/build/src-d/go-mysql-server/_integration/javascript/node_modules/mysql/lib/Connection.js␊
    - /home/travis/build/src-d/go-mysql-server/_integration/javascript/node_modules/mysql/index.js␊
    - /home/travis/build/src-d/go-mysql-server/_integration/javascript/test.js␊
    - /home/travis/build/src-d/go-mysql-server/_integration/javascript/node_modules/ava/lib/worker/subprocess.js`,
  }
```

--- a/_integration/javascript/package.json
+++ b/_integration/javascript/package.json
@@ -7,7 +7,7 @@
     "test": "./node_modules/.bin/ava"
   },
   "dependencies": {
-    "mysql": "github:mysqljs/mysql"
+    "mysql": "2.17.1"
   },
   "devDependencies": {
     "ava": "2.3.0"

